### PR TITLE
Fix overlapping timestamps on empty messages

### PR DESCRIPTION
**Pull request by goelesha**
## Notes
Fix a bug (https://github.com/vector-im/element-web/issues/21381) where I have set up min-height to avoid overlapping timestamps.
The bug occurs in Modern message layout only

## Outcome
Display before the change
Before

Display after change
Screenshot 2022-03-31 at 6 19 47 PM

Fixes: https://github.com/vector-im/element-web/issues/21381

Signed-off-by: Esha Goel <goelesha2001@gmail.com>

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Fix overlapping timestamps on empty messages ([\#8205](https://github.com/matrix-org/matrix-react-sdk/pull/8205)). Fixes vector-im/element-web#21381. Contributed by @goelesha.

Preview: https://pr8205--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: res/css/views/rooms/_EventTile.scss [hunk 1]
```diff
@@ -311,6 +311,7 @@ $left-gutter: 64px;
         .mx_EventTile_line {
             /* ideally should be 100px, but 95px gives us a max thumbnail size of 800x600, which is nice */
             margin-right: 110px;
+            min-height: 2.2rem;
```
Comment (turt2live) [inline res/css/views/rooms/_EventTile.scss:314]: This should be using one of our `$font` variables given we support font scaling.

**Review by turt2live — CHANGES_REQUESTED** [review]
Thanks for taking a look at this!

File: res/css/views/rooms/_EventTile.scss (hunk 1 above)
Comment (goelesha) [reply res/css/views/rooms/_EventTile.scss:314]: Can you pls elaborate? I didn't exactly understand what to change. As the msg is blank so their is no need to care abt the font.

Comment (turt2live) [reply res/css/views/rooms/_EventTile.scss:314]: We don't currently use `rem` in our heights and have the variables for sizing of lines broken out to `$font` variables elsewhere - if we don't use the font variables, we end up in a case where the timestamps can still overlap if the user has a high enough scale value, so we should reserve the space of what *would be* 1 line's worth of text.

Comment (goelesha) [reply res/css/views/rooms/_EventTile.scss:314]: Screenshot 2022-04-04 at 12 50 38 PM
Does this work?

Comment (turt2live) [reply res/css/views/rooms/_EventTile.scss:314]: if 14px is the variable we use normally, yes :D

Comment (goelesha) [reply res/css/views/rooms/_EventTile.scss:314]: Great. So I will commit this change then. Thanks for your review!

Comment (goelesha) [reply res/css/views/rooms/_EventTile.scss:314]: A small query, I am getting an error on line 629 although I didn't do any changes to it.
Screenshot 2022-04-04 at 1 00 22 PM
Screenshot 2022-04-04 at 1 00 34 PM

Comment (turt2live) [reply res/css/views/rooms/_EventTile.scss:314]: I think that might be your IDE trying to use a different SCSS markup than we actually use, so it errors. It shouldn't affect the CI on your PR though.

Comment (goelesha) [reply res/css/views/rooms/_EventTile.scss]: I checked the scss documentation and it says that @mixin requires curly braces inside which the properties should be specified. Like the
Screenshot 2022-04-04 at 1 08 59 PM
 one below. So I have added curly braces. Hope it doesn't create a problem.

Comment (goelesha) [reply res/css/views/rooms/_EventTile.scss]: @turt2live Is some other change required?

Comment (goelesha) [reply res/css/views/rooms/_EventTile.scss]: @turt2live Any updates on this PR?
Is any other change required from my side

**Review by turt2live — APPROVED** [review]
Thanks! Review of community PRs can take a little bit longer, unfortunately. Please don't panic if it takes us a couple weeks to get back to things :)

**Comment by turt2live** [comment]
hmm, actually, @goelesha this needs sign off as per the contributing guidelines. If that can be provided, please do so.

**Comment by goelesha** [comment]
> hmm, actually, @goelesha this needs sign-off as per the contributing guidelines. If that can be provided, please do so.

@turt2live Thanks for your review. 
I have added the sign-off in the 1st comment of this. Kindly check and pls let me know if anything else is needed.

**Comment by andybalaam** [comment]
@goelesha there are a couple of failing tests meaning this can't be merged yet. Click the Details and check what failed. If it looks irrelevant to your changes, you might just need to merge from the develop branch into your branch and `git push` again. Do ask on #element-dev:matrix.org if you need help resolving these failures.

**Comment by goelesha** [comment]
> @goelesha there are a couple of failing tests meaning this can't be merged yet. Click the Details and check what failed. If it looks irrelevant to your changes, you might just need to merge from the develop branch into your branch and `git push` again. Do ask on #element-dev:matrix.org if you need help resolving these failures.

I will look into it. Pls, give me some time.

**Review by turt2live — APPROVED** [review]
Thanks!

**Pull request merged**